### DISCUSSION TITLE
Web / Meta operating system using Etherpad

### DISCUSSION THREAD

**Discussion by titlestad** [ether/etherpad-lite#5962] [Ideas]
Hey there.

https://friendos.com - an open source web (Meta) operating system (https://github.com/friendupcloud/friendup) is now building a version of Etherpad-Lite. It runs vanilla, with client side integrations using an injected API. This will be moved to an Etherpad plugin soon - as soon as there's a clear understanding of how the plugin system works..

Anyone here who would be interested in helping out to make this version of Etherpad? Essentially, it turns Etherpad into a "desktop app" - with it's own PWA, cloud storage and collaboration infrastructure. The goal is to have it operate as a tool for school and in other collaboration scenarios, where it's part of a complete work flow.

Link to the current Friend OS app in repo. https://github.com/FriendUPCloud/friend-applications/tree/master/Etherpad

**Comment by SamTV12345** [comment]
These are exiting news indeed. Thanks for your efforts. If you have questions I do my best to answer them.

**Comment by titlestad** [comment]
Where's the best guide to get started using the HTTP and JavaScript APIs of Etherpad Lite?

I see the API reference, but not a get started guide with some examples. Does this exist?

**Reply by SamTV12345** [reply]
There is the https://etherpad.org/doc/v1.9.3/index.html#_http_api section with some examples. Does this suffice?

**Reply by titlestad** [reply]
Thank you, that's exactly what I need.
